src/algebra/algebra/spectrum.lean

```diff
@@ -167,37 +179,24 @@ end
 
 lemma inv_mem_iff {r : Rˣ} {a : Aˣ} :
   (r : R) ∈ σ (a : A) ↔ (↑r⁻¹ : R) ∈ σ (↑a⁻¹ : A) :=
-begin
-  simp only [mem_iff, not_iff_not, ←mem_resolvent_set_iff],
-  exact ⟨λ h, inv_mem_resolvent_set h, λ h, by simpa using inv_mem_resolvent_set h⟩,
-end
+not_iff_not.2 $ ⟨inv_mem_resolvent_set, inv_mem_resolvent_set⟩
 
 lemma zero_mem_resolvent_set_of_unit (a : Aˣ) : 0 ∈ resolvent_set R (a : A) :=
-by { rw [mem_resolvent_set_iff, is_unit.sub_iff], simp }
+by simpa only [mem_resolvent_set_iff, ←not_mem_iff, zero_not_mem_iff] using a.is_unit
 
 lemma ne_zero_of_mem_of_unit {a : Aˣ} {r : R} (hr : r ∈ σ (a : A)) : r ≠ 0 :=
 λ hn, (hn ▸ hr) (zero_mem_resolvent_set_of_unit a)
 
 lemma add_mem_iff {a : A} {r s : R} :
-  r ∈ σ a ↔ r + s ∈ σ (↑ₐs + a) :=
-begin
-  apply not_iff_not.mpr,
-  simp only [mem_resolvent_set_iff],
-  have h_eq : ↑ₐ(r + s) - (↑ₐs + a) = ↑ₐr - a,
-    { simp, noncomm_ring },
-  rw h_eq,
-end
+  r + s ∈ spectrum R a ↔ r ∈ spectrum R (- algebra_map R A s + a) :=
```

Comment:
Could you use notations like in the original statement. Also, since translating between the two statements would be annoying to do in the middle of a proof, I think you should keep the original statement (under a different name, the new statement matches the name better).

Reply:
I switched the order of the `↔` in the original statement and called it `add_mem_add_iff`.